src/main.rs

```diff
@@ -541,9 +627,12 @@ fn print_results_json(times: &Vec<CommitHours>) -> Result<()> {
     Ok(())
 }
 
-fn print_results(times: &Vec<CommitHours>, output_format: &OutputFormat) -> Result<()> {
+fn print_results(times: &Vec<CommitHours>, output_format: &OutputFormat, display_breakdown: &bool) -> Result<()> {
     match output_format {
-        OutputFormat::Stdout => print_results_stdout(times),
+        OutputFormat::Stdout => match *display_breakdown {
+            true => print_breakdown_results_stdout(times),
+            false => print_results_stdout(times)
+        },
         OutputFormat::Json => print_results_json(times)
```

Comment:
Is it intentional to always show the breakdown regardless of the config option in JSON mode?

Reply:
It is not at all intentional, I just failed to find a clear example how to make optional entry and having types match properly. Not very experienced with Rust... Do you have a suggestion?

Reply:
Resolved!